src/tests/timeout-handling/timeout-3a.test.ts

```diff
@@ -42,9 +42,9 @@ describe('timeout-3a extraction', () => {
     const requests = mockServer.getRequests();
     const lastRequest = requests[requests.length - 1];
 
-    // Expect last request to be emission of error event since we force-kill the worker
+    // Expect last request to be emission of error event since we force-kill the worker (using new event type string)
```

Comment:
`(using new event type string)` text will be obsolete when this is released. I'd remove these
```suggestion
    // Expect last request to be emission of error event since we force-kill the worker
```